<launch>
  <node pkg="tf" type="static_transform_publisher" name="realsense_tf" args="0 0 0 0 0 0 map RealSense_vo 100" />
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find realsense_vo_ros)/config/config_file.rviz" />

  <arg name="server" default="192.168.1.246"/>
  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
      <rosparam subst_value="true">
        server: $(arg server)
        port: 3883

        update_frequency: 100.0
        frame_id: world_optitrack

        # Use the VRPN server's time, or the client's ROS time.
        use_server_time: false
        broadcast_tf: true

        # Must either specify refresh frequency > 0.0, or a list of trackers to create
        refresh_tracker_frequency: 1.0
        #trackers:
        #- FirstTracker
        #- SecondTracker
      </rosparam>
    </node>
  <node pkg="tf" type="static_transform_publisher" name="optitrack_tf" args="0 0 0 0 0 0 map world_optitrack 100" />
  <node name="realsense_vo" pkg="realsense_vo_ros" type="real_sense_vo_node" output="screen"/>


</launch>
